minishift-ci-index.yaml

```diff
@@ -354,10 +354,23 @@
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '120m'
       # Nightly build running @ midnight daily
-      - '{git_repo}-nightly':
+      - '{git_repo}-nightly{iso_url}':
```

Comment:
@budhrg Shouldn't it will be `{git_repo}-nightly-{iso_url}`

Reply:
@agajdosi , Let's switch back to this format `{git_repo}-nightly-{iso_url}`. This will, however, impact the current `minishift-nightly` job, but we should ask CI admin to delete those jobs.

The advantage of `{git_repo}-nightly-{iso_url}` is that we will have clear picture which nightly job is using which iso when we have multiple nighly jobs.

Reply:
@praveenkumar Our plan was to use current `minishift-nightly` job, but it really looks strange in code.
@budhrg Ack, I think this will be more proper solution. Will send the updates soon.